portfolio/src/main/webapp/index.html

```diff
@@ -120,6 +130,8 @@
       <form id="data-reload" action="/data" method="POST">
         <input type="hidden" name="refresh" value="true">
       </form>
+      <form id="login-auth" action="/login" method="POST"></form>
+      <form id="logout-auth" action="/logout" method="POST" form="logout-auth"></form>
```

Comment:
Does the `form` attribute do anything when used in a `form` element? I think this shouldn't be necessary.

Reply:
Nope, good catch.